src/controllers/controllerengine.cpp

```diff
@@ -1198,7 +1198,13 @@ double ControllerEngine::getDeckRate(const QString& group) {
 
 bool ControllerEngine::isDeckPlaying(const QString& group) {
     ControlObjectThread* pPlay = getControlObjectThread(group, "play");
-    return pPlay->get() > 0.0;
+    if (pPlay) {
```

Comment:
It's better to check for error condition first, like this:

if (pPlay == NULL) {
   handle error
  return
}

return pPlay->...

That style usually results in much nicer code in general.